src/lib/score/gfx/Vulkan.cpp

```diff
@@ -55,5 +56,188 @@ QVulkanInstance* staticVulkanInstance(bool create)
 
   return g_staticVulkanInstance;
 }
+
+// Returns true if the device supports ray tracing
```

Comment:
All the vulkan code should be in the 

    #if defined(QT_FEATURE_vulkan) && QT_CONFIG(vulkan) && __has_include(<vulkan/vulkan.h>)

guard as some platforms don't have vulkan